### PR TITLE
roadmap: Proposing an initial featureset

### PR DESCRIPTION
**Pull request by kellydunn**
Adding in a proposed featureset.  Let me know what you think.  

I tried to write down all what we talked about, but I think we could do these things fairly easily with the libraries provided.

### CODE REVIEW

**Comment by flp** [comment]
Awesome. Looking good. A couple things I would add:

Deckbuilding, so that people can build a deck with their drafted cards.
Just one set of card images for initial feature set. How about BFZ?

I've never used pull-request development style. If i want to suggest changes do I comment here or make changes to your commit?

**Comment by kellydunn** [comment]
@flp I didn't fork the project, so I think you should just be able to do the following:

```
$ git fetch
$ git checkout unfiled/roadmap
// make your changes!
$ git push unfiled/roadmap
```

Alternatively, you can just comment inline on my changes by clicking the "+" button that shows up when you hover on a line number on [this page](https://github.com/flp/cuddly-quack/pull/1/files).  (this is actually how most people do it :))

File: ROADMAP.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+
+- [ ] Players 
+  - [ ] Players are anonymous by default
+  - [ ] [optional] Allow players to sign in via an Oauth token
+    - [ ] FB / Twitter / Google / w/e
+  - [ ] Players can save their draft decks in a deck list for later
+    - [ ] export as `.dek` so we can use it for other programs, like XMage, or w/e
+
+- [ ] Cards
+  - [ ] Pull images dynamically from mythicspolier.com at first as to not go against Wizard's wishes
```
Comment (flp) [inline ROADMAP.md:21]: Let's start with BFZ cards only

File: ROADMAP.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+
+- [ ] Drafting
+  - [ ] Create "Packs" consisting of 1 rare, 3 uncommons, 9 commons, and a land
+    - [ ] Provide mythic support
+    - [ ] Provide foil support
+  - [ ] Issue 3 "Packs" randomly to all members of the draft
+  - [ ] Exchange packs player by player in Left, Right, Left format
+    - [ ] Implement a timer so that players are forced to choose within the apropriate time limit
+    - [ ] Implement an auto choose function when they run out of time
+    - [ ] Pass remainder of the pack to the person next in line
```
Comment (flp) [inline ROADMAP.md]: New section: deckbuilding so that players can create a deck out of their draft

**Pull request merged**